imperial_coldfront_plugin/views.py

```diff
@@ -1 +1,59 @@
 """Plugin views."""
+
+from django.conf import settings
+from django.core.mail import send_mail
+from django.core.signing import BadSignature, SignatureExpired, TimestampSigner
+from django.http import HttpRequest, HttpResponse, HttpResponseBadRequest
+from django.shortcuts import render
+from django.urls import reverse
+
+
+def invite_to_group(request: HttpRequest) -> HttpResponse:
+    """Add an individual to a group."""
+    signer = TimestampSigner()
+
+    # TODO: Get invitee email from request.
+    invitee_email = "my@email.org"
+
+    # Sign invitation.
+    token = signer.sign_object(
+        {
+            "inviter_pk": request.user.pk,
+            "invitee_email": invitee_email,
+        }
+    )
+
+    # Send invitation via email.
+    # TODO: get_host does not include protocol -- need an automagic way to get it.
+    invite_url = "http://" + request.get_host() + reverse("accept_invite", args=[token])
```

Comment:
You can use [HttpRequest.build_absolute_uri](https://docs.djangoproject.com/en/5.1/ref/request-response/#django.http.HttpRequest.build_absolute_uri).